.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -20,7 +20,7 @@ body:
   - type: input
     attributes:
       label: Version of BioDrop (for example "v1.2.3")
-      description: Can be found in the lower right corner of the web interface in the footer.
+      description: Can be found in the footer of the BioDrop.io website.
```

Comment:
I think it would be better to give a precise location. What do you think of this!?

```suggestion
      description: Check the lower left corner of the BioDrop website's footer for version info.
```

Reply:
Hi, I did think of the same thing, so I asked for opinions on discord. This was Emma's suggestion and we eventually agreed upon it.
![image](https://github.com/EddieHubCommunity/BioDrop/assets/71211731/6fe7330d-7d86-4d88-9d6f-5f602fc58cbc)